### PR TITLE
[ChatGPT] fix some bug

### PR DESCRIPTION
**Pull request by bxb100**
## Description

- [fix: proxy not working](https://github.com/raycast/extensions/commit/1d068d17690d16434bc2d96f446a4adb864f0e87)
- [fix](https://github.com/raycast/extensions/commit/8d400f30b1d9c9d1eef0742a8c1c1e174afa79eb)[:](https://github.com/raycast/extensions/commit/8d400f30b1d9c9d1eef0742a8c1c1e174afa79eb) https://github.com/raycast/extensions/issues/9146 [model fetch lag](https://github.com/raycast/extensions/commit/8d400f30b1d9c9d1eef0742a8c1c1e174afa79eb)
- [fix](https://github.com/raycast/extensions/commit/726eb6a3f48e5830d9e8bfefb27de1d9c92ce9ac)[:](https://github.com/raycast/extensions/commit/726eb6a3f48e5830d9e8bfefb27de1d9c92ce9ac) https://github.com/raycast/extensions/issues/10391
- add some lint [chore: lint fix](https://github.com/raycast/extensions/commit/76de8d89df52df73126cae2e7c68f6c33973ace3)

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/chatgpt/.eslintrc.json [hunk 1]
```diff
@@ -1,6 +1,16 @@
...
+      "error",
+      {
+        "newlines-between": "always",
+        "alphabetize": {
+          "order": "asc"
+        }
+      }
+    ]
+  }
+}
```
Comment (pernielsentikaer) [inline extensions/chatgpt/.eslintrc.json:16]: Why change this file back to an older version?

Comment (bxb100) [reply extensions/chatgpt/.eslintrc.json:16]: oh. sry, I will revert this

Comment (bxb100) [reply extensions/chatgpt/.eslintrc.json]: done

**Review by pernielsentikaer — COMMENTED** [review]
I just tested the `Models` command, I'll end here without anything else happening, do you know why, it never stops loading

Raycast29012024-zfS4f5kN png@2x

**Review by pernielsentikaer — APPROVED** [review]
It seems to work better now, let's go with that and see how it goes 🙂

**Pull request merged**